### PR TITLE
Add space after Kotlin keywords so that you can chain in keyword commands

### PR DESCRIPTION
**Pull request by jaresty**

### CODE REVIEW

File: lang/kotlin/kotlin.py [hunk 1]
```diff
@@ -8,19 +8,19 @@
...
+    "public ": "public",
+    "private ": "private",
+    "protected ": "protected",
+    "static ": "static",
+    "synchronized ": "synchronized",
+    "volatile ": "volatile",
+    "transient ": "transient",
+    "abstract ": "abstract",
+    "interface ": "interface",
+    "final ": "final",
```
Comment (auscompgeek) [inline lang/kotlin/kotlin.py:23]: These are on the wrong side: the keys are the spoken phrase, the values will be what's inserted.

**Review by auscompgeek — CHANGES_REQUESTED** [review]

File: lang/kotlin/kotlin.py (hunk 1 above)
Comment (jaresty) [reply lang/kotlin/kotlin.py]: Thanks for pointing that out. Should be fixed now.

**Review by auscompgeek — APPROVED** [review]
Looks good. I believe this was the intended way to use this list.

**Comment by jaresty** [comment]
can someone help me with merging this? I don't have write access so I'm unable to merge it on my own.

**Pull request merged**